arrows/core/initialize_cameras_landmarks.cxx

```diff
@@ -182,8 +187,6 @@ class initialize_cameras_landmarks::priv
   vital::algo::bundle_adjust_sptr bundle_adjuster;
   /// Logger handle
   vital::logger_handle_t m_logger;
-public:
-  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
I think if we take one of these out, we really should be taking all of them out. Right @billhoffman ?